ClientApp/services/eventService.ts

```diff
@@ -0,0 +1,12 @@
+import axiosInstance from '@/services/axiosInstance';
+import { API_ENDPOINTS } from '@/utils/api/endpoints';
+//API_ENDPOINTS.CREATE_EVENT
+export const createEvent = async (eventData: any) => {
+  try {
+    const response = await axiosInstance.post("https://api-dev.sportahub.app/api/event-service/event", eventData);
```

Comment:
We should avoid putting the entire URL here. I think you forgot to change it here and use the API_ENDPOINTS file.

Reply:
agreed!